### ISSUE TITLE
Admin Delete and Reset Achievements Feature

### ISSUE THREAD

**Issue by SirCraftypants** [PyvesB/AdvancedAchievements#816]
:clipboard: **Description**

Ability for Admin to click and individual Achievement of any player to delete or reset

left click to delete
right click to reset
I don't know any Devs personally and have no experience in Developing, if i could i would definitely help add it. Now if this feature, an Admin/Op only permission and the ability to add Lore: to Achievements was added I will make a Donation. Or make a premium version with these added and i'll buy it.

**Comment by PyvesB** [comment]
Hello @SirCraftypants,

These features already exist, using the _/aach delete_ and _/aach delete commands. See the [Wiki](https://github.com/PyvesB/AdvancedAchievements/wiki/Commands-and-permissions#commands) for more information. :wink:

**Comment by SirCraftypants** [comment]
Yes but i wondered if you could make it so that clicking on them with right click and left click for only admin. I aim for a large population on my server and thought it would make it a lot easier for admins rather than constantly typing in the commands.

**Comment by PyvesB** [comment]
I'm not following the reasoning, it definitely does not feel "a lot easier" to me, on the contrary.
* currently it's a single command to delete an achievement or reset a category. Both commands have autocompletion for all parameters and support * wildcards.
* with a GUI, you would need a first command to open the GUI for a specific player. You would then have to click on the achievement category you're interested in. You then have to find the right achievement or sub-category. That's potentially several more clicks to browse and search through the pagination system, which can be quite tedious if you've got many achievements. Once you've found what you want to delete/reset, it's yet another click which you can easily get wrong if you invert left and right click.

As a summary, one the one hand it's one command, on the other it's one command plus a minimum of two error-prone clicks and some potentially tedious searching.

Furthermore, the GUI wouldn't work from the server's console, which means you can't apply this approach to large player populations; with _/aach delete_ and _/aach reset_, you can easily prepare thousands of commands with some simple copy-paste-replace in a text editor and apply mass changes in one go. 😉
